Build fails with: g++: error: PROTOBUF_USE_DLLS: linker input file not found:
No such file or directory

This is because 'PROTOBUF_USE_DLLS' is added bare to the g++ invocation. The
reason why is not clear.

Partially-reverts: https://github.com/netdata/netdata/commit/2a04a06569cec70da71d04a58954a030384b4cf7
diff --git a/packaging/cmake/Modules/NetdataProtobuf.cmake b/packaging/cmake/Modules/NetdataProtobuf.cmake
index 48be402..d4ae3ae 100644
--- a/packaging/cmake/Modules/NetdataProtobuf.cmake
+++ b/packaging/cmake/Modules/NetdataProtobuf.cmake
@@ -219,7 +219,7 @@ endfunction()
 
 # Add protobuf to a specified target.
 function(netdata_add_protobuf _target)
-        target_compile_options(${_target} PRIVATE ${NETDATA_PROTOBUF_CFLAGS_OTHER})
+        target_compile_definitions(${_target} PRIVATE ${NETDATA_PROTOBUF_CFLAGS_OTHER})
         target_include_directories(${_target} PRIVATE ${NETDATA_PROTOBUF_INCLUDE_DIRS})
         target_link_libraries(${_target} PRIVATE ${NETDATA_PROTOBUF_LIBS})
 endfunction()
